app/models/friendship.rb

```diff
@@ -0,0 +1,6 @@
+class Friendship < ApplicationRecord
+  belongs_to :user
+  belongs_to :friend, class_name: 'User'
+  validates :user, presence: true
+  validates :friend, presence: true
```

Comment:
- User model needs: ```has_many :friendships```

Reply:
I think we implemented this the User model actually has_many friendships. I cannot see where the problem is. Can you please give more explanation?